Dockerfile

```diff
@@ -1,10 +1,12 @@
 FROM quay.io/ukhomeofficedigital/docker-centos-base
 
+ENV NODE_VERSION v4.2.2
```

Comment:
I changed it but honestly I'd rather the file were more reable than efficant when it comes to versions like that.